x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/append.tsx

```diff
@@ -52,7 +52,12 @@ export const Append: FunctionComponent = () => {
         })}
       />
 
-      <UseField config={fieldsConfig.value} component={ComboBoxField} path="fields.value" />
+      <UseField
+        data-test-subj="valueFieldAppend"
```

Comment:
can we rename this to `valueField`? I think you and I may have renamed it to `valueFieldAppend` when we were troubleshooting a test failure together, but I don't think it's necessary. (If there is a naming conflict, perhaps `appendValueField` would be better.)

Reply:
Fixed.